flows/actions/base.go

```diff
@@ -297,11 +297,7 @@ func resolveGroups(run flows.FlowRun, references []*assets.GroupReference, stati
 		}
 
 		if group != nil {
-			if staticOnly && group.IsDynamic() {
```

Comment:
nicely there's no duplication required.. we can just move this down to the modifier